src/unstract/adapters/llm/vertex_ai/src/vertex_ai.py

```diff
@@ -39,7 +40,7 @@ class SafetySettingsConstants:
 
 
 UNSTRACT_VERTEX_SAFETY_THRESHOLD_MAPPING: dict[str, HarmBlockThreshold] = {
-    "HARM_BLOCK_THRESHOLD_UNSPECIFIED": HarmBlockThreshold.HARM_BLOCK_THRESHOLD_UNSPECIFIED,
+    "HARM_BLOCK_THRESHOLD_UNSPECIFIED": HarmBlockThreshold.HARM_BLOCK_THRESHOLD_UNSPECIFIED,  # noqa: E501
```

Comment:
What does noqa: E501 do? Line width check skip? @chandrasekharan-zipstack

Reply:
Yes @gaya3-zipstack, did it for the dict to remain readable